pnpm-lock.yaml

```diff
@@ -20958,6 +20947,12 @@ packages:
     hasBin: true
     dev: true
 
+  /prettier/2.8.8:
```

Comment:
I noticed prettier 2.5.1 is still listed above.

Lets also update the other dependencies to fix this:

```
cd ./packages/create-next-app
pnpm add -D prettier-plugin-tailwindcss@latest
```

cc @balazsorban44 who added that in https://github.com/vercel/next.js/pull/46927

Reply:
Done!